chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.erb

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# This script starts the upgrade of the node server to the latest version of
+# system and Cloud product.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+mkdir -p "`dirname "$LOGFILE"`"
+
+# Copy stdout to fd 3
+exec 3>&1
```

Comment:
This part is a bit confusing. First line redirects fd3 to fd1, which is different from what comment says. Also it is not clear where fd3 is used.

Also, may be it's clearer to redirect fd1 and fd2 to fd4 instead of fd1 to fd4 and fd2 to fd1.